agent/doc/swdesign/README.md

```diff
@@ -543,6 +543,27 @@ Needs:
 - impl
 - utest
 
+#### Control Interface created for eligible workloads
+`swdd~agent-control-interface-optional-creation~1`
```

Comment:
Maybe we should also change the Id according to the name 😉

Reply:
Done.